### PR TITLE
feat: Sort by relevance score if available

### LINKED ISSUE #537: Rank results by full text score from sources
Investigate whether we can return matching scores from sources that support them to increase result relevance for users.

Previously, we decided to sort results alphabetically because we cannot guarantee the order of results (#68). However, if the source supports a score property, we can sort on that, even client-side if necessary.

* For Virtuoso, this should be `?o <bif:contains> "gogh" OPTION (score ?sc)`, but see https://github.com/comunica/comunica/issues/940.
* Wikidata’s `mwapi` [doesn’t seem to support scores](https://www.mediawiki.org/wiki/Wikidata_Query_Service/User_Manual/MWAPI#Examples).
* …

### PR DESCRIPTION
**Pull request by ddeboer**
* Queries may sort by relevance score.
* If they do, sort by score primarily. For identical scores, fall back to alphabetical sort.
* If they do not, only sort alphabetically.
* This PR does not yet expose the score in the GraphQL API.

Fix #537

### CODE REVIEW

File: packages/network-of-terms-query/src/terms.ts [hunk 1]
```diff
@@ -55,6 +57,7 @@ export class TermsTransformer {
     ['http://www.w3.org/2004/02/skos/core#related', 'relatedTerms'],
     ['http://www.w3.org/2008/05/skos#related', 'relatedTerms'],
     ['http://www.w3.org/2004/02/skos/core#inScheme', 'inScheme'],
+    ['http://jena.apache.org/text#score', 'score'],
```
Comment (ddeboer) [inline packages/network-of-terms-query/src/terms.ts:60]: Of course, this is not the namespace that we want to use for storing the score in queries. It should be decoupled from any particular SPARQL store.

@EnnoMeijers Do you have ideas for a generic RDF vocabulary concerning search results and their relevance scores?

Comment (EnnoMeijers) [reply packages/network-of-terms-query/src/terms.ts:60]: Hmm, good question! Maybe use http://purl.org/voc/vrank, it seems to be around for quite some time, the vrank:rankValue property could be appropriate? Vrank has also properties to describe the ranking method in more detail.

Comment (ddeboer) [reply packages/network-of-terms-query/src/terms.ts:60]: Good suggestion, thanks!

**Pull request merged**